Resources/Prototypes/Entities/Structures/Doors/Windoors/base.yml

```
@@ -10,7 +10,7 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-        bounds: "-0.2,-0.49,-0.49,0.49"
+        bounds: "-0.49,-0.49,0.49,-0.2"
```

same question as above
```suggestion
        bounds: "-0.49,-0.2,0.49,-0.49"
```